pepper-apis/src/main/java/org/broadinstitute/ddp/filter/StudyLanguageResolutionFilter.java

```diff
@@ -34,7 +34,7 @@
     private static final String INDONESIAN_NEW = "id";
 
     private static final Logger LOG = LoggerFactory.getLogger(StudyLanguageResolutionFilter.class);
-    private static final String STUDY_GUID_REGEX = "/studies/(\\w+)";
+    private static final String STUDY_GUID_REGEX = "/studies/([0-9a-zA-Z-]+)";
```

Comment:
Original regex did not properly parse studies that have a `-` in the name, e.g. `cmi-brain`.